index.md

```diff
@@ -4,23 +4,41 @@ layout: default
 
 _“Best practice templates and writing instructions for documenting open source software.”_
 
+### Why?
 
-### Download
+We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. In the process, we expect to lift the quality of technical writing for all software.
 
-* The [0.1 alpha straw-man release of templates](https://github.com/thegooddocsproject/templates/releases/tag/v0.1) was released at the 2019 Write The Docs - Australia conference.
-* The latest development version of templates are available from the [github main branch](https://github.com/thegooddocsproject/templates).
+### How?
 
-### Why?
+We are creating processes, templates, and guides covering the numerous aspects of writing good documentation. We describe our reasoning, referencing research where available, to help writers decide when they should deviate from suggestions.
+
+### What?
 
-Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. How do you structure docs? What does a docs website look like? How do I keep docs consistently formatted? The list of questions far outweigh the answers.
+Every project deserves a minimum level of _Good Docs_, but it can be really hard to know where to start or what to write. Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of documentation using a suite of templates.
 
```

Comment:
I love all your changes! I'm curious if we have defined our MVD yet. If so, perhaps a link would help here.

Reply:
Oh man, yeah. I was curious about this sentence too. I don't think we've used this phrase anywhere else, and I don't think we've defined the MVD, but I could be wrong. 
Hmm, how about "Our open source suite of templates will help you create a baseline set of documentation."